db/schema.rb

```diff
@@ -336,6 +337,13 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "whitehall_imported_assets", force: :cascade do |t|
+    t.bigint "document_id", null: false
+    t.string "asset_manager_url", null: false
+    t.string "new_url"
```

Comment:
You probably won't be able to do this as we won't know the URL until we've synced with Asset Manager - might be better to save a reference to the revision and variant.